### ISSUE TITLE
Remove Illustration

### ISSUE THREAD

**Issue by warpech** [StarcounterSamples/Simplified#14]
In the current data model, [`Something`](https://github.com/StarcounterSamples/Simplified/blob/fb6b1e2299839fd9f60dced7165ff2f0659244bb/Ring1/Something.cs) has a property [`Illustration`](https://github.com/StarcounterSamples/Simplified/blob/fb6b1e2299839fd9f60dced7165ff2f0659244bb/Ring1/Illustration.cs).

I think that it is wrong because it makes the concept of `Illustration` special and gives unfair advantage to the "Images" app. What I mean is that giving an **image** to something is not the same as giving a **barcode** to something. And it should be exactly the same.

My proposal is to remove `Simplified.Ring1.Illustration` and `Simplified.Ring1.Content`. Instead, create new classes `Simplified.Ring2.Image` and `Simplified.Ring2.SomethingsImage : Relation`.

After that (or maybe even **before** that, for testing?), `Barcode` should be modelled the same way. Let's have `Barcode` and `SomethingsBarcode : Relation`. We can then remove the `Barcode` property from `Product`.

@miyconst, @Starcounter-Jack WDYT?

**Comment by miyconst** [comment]
Positive. The `Simplified.Ring1.Illustration` confused me a couple of time.

The only thing I would change is the names: `Something_Barcode`, `Something_Image`, means without the `s` letter, not with an underscore :).

**Comment by aloscha** [comment]
And the `Simplified.Ring2.Image` will have fields:
- Content (Url)
- Something?

**Comment by warpech** [comment]
Thanks!

> The only thing I would change is the names: Something_Barcode, Something_Image, means without the s letter, not with an underscore :).

I was inspired by [`SomebodiesRelation.cs`](https://github.com/StarcounterSamples/Simplified/blob/master/Ring1/SomebodiesRelation.cs) and wanted to be consistent with it.

**Comment by miyconst** [comment]
>And the `Simplified.Ring2.Image` will have fields:

No, it won't have property `Something`. The relation between Images and Somethings will made via `SomethingsImage` class.

**Comment by miyconst** [comment]
>I was inspired by `SomebodiesRelation.cs` and wanted to be consistent with it.

I also don't like this name, but it's fine. Let's be consistent and keep the `'s`.

**Comment by aloscha** [comment]
@miyconst and what about that naming?
`  Handle.POST("/**images/images**", (Request request) => {`

**Comment by miyconst** [comment]
@aloscha sorry I don't get your question.

**Comment by aloscha** [comment]
I mean if will create/change names I also suggest to change some uri like /images/images, which tell nothing.

**Comment by warpech** [comment]
> I mean if will create/change names I also suggest to change some uri like /images/images, which tell nothing.

Well, I agree but this has nothing to do with this issue.

The way I see it:

```cs 
    public class Image : Simplified.Ring1.Something {
        public string URL;
    }

    public class SomethingsImage : Simplified.Ring1.Relation {
        [SynonymousTo("WhatIs")]
        public Image Image;
    }
```
